pyproject.toml

```diff
@@ -0,0 +1,115 @@
+[tool.mypy]
+# Be vewy, vewy stwict, we'we hunting wabbits^Wbugs.
+strict = true
+
+# Automagically make imported object part of mypy's view of module's namespace.
+implicit_reexport = true
+
+# Make error messages more rich, this helps better targeted waivers.
+show_error_codes = true
+
+# Allow mypy to follow imports to get correct picture of type
+# annotations across module boundaries.
+follow_imports = "normal"
+
+python_version = "3.7"
+files = ["bin/tmt", "tmt/", "setup.py"]
+
+[[tool.mypy.overrides]]
```

Comment:
Just curious: Why are double brackets used here while all other sections are single bracket, including the subsection `tool.ruff.per-file-ignores`.

Reply:
It's how mypy [wants](https://mypy.readthedocs.io/en/stable/config_file.html#using-a-pyproject-toml-file) it to be declared. 
Using TOML's [array of tables](https://toml.io/en/v1.0.0#array-of-tables).  
For example, [cibuiltwheel](https://cibuildwheel.readthedocs.io/en/stable/options/#overrides) also uses it for their "override system".